ios/sdk/src/JitsiMeetConferenceOptions.m

```diff
@@ -76,11 +89,14 @@ - (BOOL)videoMuted {
 }
 
 - (void)setWelcomePageEnabled:(BOOL)welcomePageEnabled {
-    _welcomePageEnabled = [NSNumber numberWithBool:welcomePageEnabled];
+    [self setFeatureFlag:WelcomePageEnabledFeastureFlag
```

Comment:
a typo in `Feasture`

Reply:
Stupid Xcode refactoring thing, didn't change all occurences! Fixed.